std_msgs/Header header
geometry_msgs/Point position
int32 num_visible_markers
